stdlib/all_jlls.version

```diff
@@ -0,0 +1,28 @@
+## All `_jll` names and versions
```

Comment:
If this is supposed to be a Makefile, please add
```suggestion
# -*- mode: make -*-
## All `_jll` names and versions
```
so we get the right syntax highlighting (and Emacs users get the right mode).  Same for all the other `*.version` files.

Reply:
Yes this gets included in other makefiles, hence is a makefile.